GitUI/CommandsDialogs/FormBrowse.Designer.cs

```diff
@@ -528,6 +530,17 @@ private void InitializeComponent()
             this.toolStripSeparator2.Name = "toolStripSeparator2";
             this.toolStripSeparator2.Size = new System.Drawing.Size(6, 25);
             // 
+            // toolStripFileExplorer
+            // 
+            this.toolStripFileExplorer.CheckOnClick = true;
+            this.toolStripFileExplorer.Enabled = false;
+            this.toolStripFileExplorer.Image = global::GitUI.Properties.Resources.Folder;
+            this.toolStripFileExplorer.ImageTransparentColor = System.Drawing.Color.Gray;
+            this.toolStripFileExplorer.Name = "toolStripFileExplorer";
+            this.toolStripFileExplorer.Size = new System.Drawing.Size(23, 22);
+            this.toolStripFileExplorer.ToolTipText = "File Explorer";
```

Comment:
this text needs to be translated

the easiest way is probably do the following:

```
        private FormBrowse()
        {
            InitializeComponent();
            Translate();

            toolStripFileExplorer.ToolTipText = fileExplorerToolStripMenuItem.Text;
        }
```

@jbialobr what do you think?

Reply:
ToolTipTexts assigned this way are translated automagically. https://github.com/gitextensions/gitextensions/issues/3745

Reply:
For the automagic to work, someone will still need to provide translations, won't they?
`fileExplorerToolStripMenuItem` performs exactly the same function and has the required text, which is already translated. Hence my suggestion to copy and assign the existing translation to the new element.

...Or does the automagic work somehow differently?

Reply:
>For the automagic to work, someone will still need to provide translations, won't they?

Yes, that is true. But I would not mix `Text` and `ToolTipText`. It so happened they share the same text, but it should be handled on the translation side to prompt the same translation for the same texts when a translator is translating.

Reply:
Thanks @RussKie and @jbialobr.  RussKie, do you want me to make any changes here? 

P.S the other requested change has been committed